internal/pkg/skuba/deployments/ssh/states.go

```diff
@@ -32,12 +32,12 @@ type Runner func(t *Target, data interface{}) error
 
 func (t *Target) Apply(data interface{}, states ...string) error {
 	for _, stateName := range states {
-		klog.V(1).Infof("=== applying state %s ===", stateName)
+		klog.Infof("=== applying state %s ===", stateName)
 		if state, stateExists := stateMap[stateName]; stateExists {
 			if err := state(t, data); err != nil {
 				return errors.Wrapf(err, "failed to apply state %s", stateName)
 			} else {
-				klog.V(1).Infof("=== state %s applied successfully ===", stateName)
+				klog.Infof("=== state %s applied successfully ===", stateName)
```

Comment:
[1]

Reply:
Hmm, this change  is for another bug. I wanted to make in one shot as a logging stuff.
This change comes from  https://bugzilla.suse.com/show_bug.cgi?id=1140526.